### ISSUE TITLE
RuleTester prints `\u000a` instead of newline

### ISSUE THREAD

**Issue by golopot** [eslint/eslint#12027]
Since eslint 6, rule tester prints newline in codes as `\u000a`. It is hard to read for plugin developers. 

#### Code to repoduce
```
npx mocha index.js
```

```js
// index.js

const {RuleTester} = require('eslint');

const someRule = {
  create() {
    return {};
  },
};

const ruleTester = new RuleTester();

ruleTester.run('some-rule', someRule, {
  valid: [],
  invalid: [
    {
      code: `
        var a;
        var b;
      `,
      errors: [{message: 'foooooooooooo'}],
    },
  ],
});
```
#### Actual output
Note the lines with `\u000a`.
```
  0 passing (46ms)
  1 failing

  1) some-rule
       invalid
         \u000a        var a;\u000a        var b;\u000a      :

      AssertionError [ERR_ASSERTION]: Should have 1 error but had 0: []
      + expected - actual

      -0
      +1
```

#### Expected ouput
This is the behavior with `npm i eslint@5`.
```
  1) some-rule
       invalid
         
        var a;
        var b;
      :

      AssertionError [ERR_ASSERTION]: Should have 1 error but had 0: []
      + expected - actual

      -0
      +1
```

**Comment by zamboney** [comment]
I would like it to be my first issue

**Comment by platinumazure** [comment]
Hi @golopot, thanks for the issue.

Based on your description, this does sound like a bug. Hopefully a team member can reproduce this bug soon so we can mark the issue as accepted. (I'm on vacation without access to my PC, so I can't test this myself right now.)

**Comment by ulrichb** [comment]
Same here:

![image](https://user-images.githubusercontent.com/388796/62248860-67b24900-b3e9-11e9-891f-78f62f220185.png)

(BTW: Running on Windows.)

**Comment by golopot** [comment]
An example in ci https://travis-ci.org/yannickcr/eslint-plugin-react/jobs/573750631

**Comment by platinumazure** [comment]
I'll mark as accepted based on the CI example. Thanks!
